runtime-modules/content/src/benchmarks/benchmarking.rs

```diff
@@ -3249,6 +3249,154 @@ benchmarks! {
+    //   - all feature paused (except necessary ones for extr to succeed)
+    //   - max channel assets
+    //   - max collaborators
+    // INPUT COMPLEXITY
+    // - remark message byte-length: b
+    channel_agent_remark {
+        let b in 1 .. MAX_BYTES_METADATA;
+        let (channel_id, group_id, lead_account_id, curator_id, curator_account_id) =
+            setup_worst_case_scenario_curator_channel_all_max::<T>(false)?;
+        let origin = RawOrigin::Signed(curator_account_id);
+        let actor = ContentActor::Curator(group_id, curator_id);
+        let msg = vec![1u8].repeat(b as usize);
+    }: _(origin, actor, channel_id, msg.clone())
+        verify {
+            assert_last_event::<T>(
+                <T as Config>::Event::from(
+                    Event::<T>::ChannelAgentRemarked(
+                        actor,
+                        channel_id,
+                        msg,
+                    )
+                ).into()
+            );
+        }
+
+    // WORST CASE SCENARIO
+    // STATE COMPLEXITY
+    // - curator owned channel
+    // - channel-owning curator group has max number of permissions per level
+    // - curator number is max
+    // - curator has max number of agent permissions
+    // - channel has max size:
+    //   - all feature paused (except necessary ones for extr to succeed)
+    //   - max channel assets
+    //   - max collaborators
+    // - video has max size
+    //   - max video assets
+    // INPUT COMPLEXITY
+    // - remark message byte-length: b
+    nft_owner_remark {
+        let b in 1 .. MAX_BYTES_METADATA;
+        let (
+            video_id,
+            (curator_account_id, actor, channel_id, _)
+        ) = setup_worst_case_scenario_mutable_video::<T>(
+            Some(T::MaxNumberOfAssetsPerVideo::get()),
+            T::StorageBucketsPerBagValueConstraint::get().max() as u32,
+        )?;
+
+        let (nft_owner_actor, owner_account) = setup_idle_nft::<T>(
+            curator_account_id,
+            actor,
+            video_id,
+            false,
+        )?;
+
+        let origin = RawOrigin::Signed(owner_account);
+        let msg = vec![1u8].repeat(b as usize);
+    }: _(origin, nft_owner_actor, video_id, msg.clone())
+        verify {
+            assert_last_event::<T>(
+                <T as Config>::Event::from(
+                    Event::<T>::NftOwnerRemarked(
+                        nft_owner_actor,
+                        video_id,
+                        msg
+                    )
+                ).into()
+            );
+        }
+
+    // WORST CASE SCENARIO
+    // COMPLEXITY
+    // DB OPERATIONS
+    // - DB Write: Balance
+    // - DB Read: Balance
+    update_channel_state_bloat_bond {
+        let (_, _, lead_account_id, _, _) =
+            setup_worst_case_scenario_curator_channel_all_max::<T>(false)?;
+        let origin = RawOrigin::Signed(lead_account_id);
+        let new_channel_bloat_bond: BalanceOf::<T> = 100u32.into();
```

Comment:
Should be at least `T::ExistentialDeposit::get()` (this condition was recently added in `carthage`)